gettsim/incomes.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+import pandas as pd
+
+from gettsim.func_out_columns import DPI
+from gettsim.func_out_columns import GROSS
+
+
+def disposible_income(df):
+    disp_inc = pd.DataFrame(index=df.index, columns=DPI)
+    disp_inc["dpi_ind"] = df[
+        [
+            "m_wage",
+            "m_kapinc",
+            "m_self",
+            "m_vermiet",
+            # "m_imputedrent", Do we need this?
```

Comment:
Whether one "needs" imputed rents is a matter of conventions. Note though that if you want to align to "official" inequaliy aggregates, imputed rents need to be added

Reply:
But it is nowhere else used?

Reply:
We haven't decided yet on what's mandatory input and what's not.

Reply:
Then we should decide at some point and in the same move also which variables we compute inside the functions. I'll put that in a separate issue!